src/libslic3r/PrintConfig.cpp

```diff
@@ -4406,17 +4406,17 @@ void PrintConfigDef::init_fff_params()
     def = this->add("support_type", coEnum);
     def->label = L("Type");
     def->category = L("Support");
-    def->tooltip = L("normal(auto) and tree(auto) is used to generate support automatically. "
-                     "If normal(manual) or tree(manual) is selected, only support enforcers are generated");
+    def->tooltip = L("Normal (auto) and Tree (auto) is used to generate support automatically. "
+                     "If Normal (manual) or Tree (manual) is selected, only support enforcers are generated");
     def->enum_keys_map = &ConfigOptionEnum<SupportType>::get_enum_values();
-    def->enum_values.push_back("normal(auto)");
-    def->enum_values.push_back("tree(auto)");
-    def->enum_values.push_back("normal(manual)");
-    def->enum_values.push_back("tree(manual)");
-    def->enum_labels.push_back(L("normal(auto)"));
-    def->enum_labels.push_back(L("tree(auto)"));
-    def->enum_labels.push_back(L("normal(manual)"));
-    def->enum_labels.push_back(L("tree(manual)"));
+    def->enum_values.push_back("Normal (auto)");
+    def->enum_values.push_back("Tree (auto)");
+    def->enum_values.push_back("Normal (manual)");
+    def->enum_values.push_back("Tree (manual)");
+    def->enum_labels.push_back(L("Normal (auto)"));
+    def->enum_labels.push_back(L("Tree (auto)"));
+    def->enum_labels.push_back(L("Normal (manual)"));
+    def->enum_labels.push_back(L("Tree (manual)"));
```

Comment:
Don't update values, update labels only, otherwise all existing profiles/projects will break.

Reply:
Thanks for the feedback, I'll revise this.